docker-compose.yml

```diff
@@ -127,6 +128,20 @@ services:
       # If you have high volume and your search return incomplete results
       # You might want to change this to a higher value (and ensure your host has enough memory)
       MAX_MEMORY_USAGE_RATIO: 0.3
+  geoipupdate:
+    << : *restart_policy
+    image: maxmindinc/geoipupdate
+    environment:
+      GEOIPUPDATE_FREQUENCY: 168  # one week, in hours
+      # Set the following (e.g., via docker-compose.override.yml) to enable IP geolocation.
```

Comment:
With the current definition, they can pass these as environment variables too. `docker-compose` will inherit the value set in the environment already.

Reply:
Yup. Hence the "e.g." ... override.yml is one way to do it. I think it's the way we should recommend because otherwise people have to come up with some other means of managing envvars, which is complex and highly idiosyncratic.

Reply:
I think for these kind of "secret"s, it is better to pass these in via env variables? Also we have not advertised `docker-compose.override.yml` yet. I know you are going to add that to the docs, which is great, but I think we need some homework to do before that.

See #694, or #698 for instance.

Reply:
Remove reference to `docker-compose.override.yml` in f47e60c0b7c26ae92360c567944041d1eaea2695.